src/main/resources/META-INF/plugin.xml

```diff
@@ -310,6 +310,13 @@
                              language="textmate"
                              implementationClass="com.redhat.devtools.lsp4ij.features.foldingRange.LSPFoldingRangeBuilder"
                              order="first"/>
+        <!-- The code block providers are based on the same information from textDocument/folding -->
+        <codeBlockProvider language="TEXT"
```

Comment:
`codeBlockProvider`s are registered for both plain text and TextMate.

Reply:
Please update the doc at https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPSupport.md#folding-range with codeBlock extension point sionce it is working by default only for TEXT and textmate

Reply:
Ah, you mean specifically the `codeBlockProvider` extension point. I get it now. I'll update the docs. I'll also include an updated demo using TypeScript. This will all be in a new, distinct PR.

Reply:
See #676.